src/S3/MultipartCopy.php

```diff
@@ -51,24 +52,25 @@ class MultipartCopy extends AbstractUploadManager
      *   result of executing a HeadObject command on the copy source.
      *
      * @param S3ClientInterface $client Client used for the upload.
-     * @param string            $source Location of the data to be copied
+     * @param string $source Location of the data to be copied
      *                                  (in the form /<bucket>/<key>).
-     * @param array             $config Configuration used to perform the upload.
+     * @param array $config Configuration used to perform the upload.
      */
     public function __construct(
         S3ClientInterface $client,
         $source,
         array $config = []
-    ) {
+    )
+    {
         if (ArnParser::isArn($source)) {
             $this->source = '';
         } else {
             $this->source = "/";
         }
         $this->source .= ltrim($source, '/');
         parent::__construct($client, array_change_key_case($config) + [
-            'source_metadata' => null
-        ]);
+                'source_metadata' => null
```

Comment:
Style nitpick: Can this fit on one line? If not, each parameter should be on its own line:

```
parent::__construct(
    $client, 
    array_change_key_case($config) + [ 'source_metadata' => null]
);
```

Reply:
No, it's 88 characters.  I made the switch to separate the parameters